finish/src/main/java/io/openliberty/guides/inventory/InventoryManager.java

```diff
@@ -28,14 +31,17 @@
 public class InventoryManager {
 
   private InventoryList invList = new InventoryList();
-  private SystemClient systemClient = new SystemClient();
+  private InventoryUtils invUtils = new InventoryUtils();
+
+  @Inject
+  @RestClient
+  private SystemClient defaultRestClient;
```

Comment:
is this `defaultRestClient` used anywhere? From our talk a while back, it doesn't seem like any of the guides require a defaultRestClient, they'll all build a rest client using RC builder instead. So I think this bit should be here.

Reply:
just to clear things up, as we discussed during the meeting yesterday, the default RC was needed in the RC guide in order to demonstrate all of the ways in which a RC can be created. This is not necessary for other guides as they can just stick to one of these ways, which would be RCBuilder